advanced/Scripts/piholeDebug.sh

```diff
@@ -230,13 +230,17 @@ copy_to_debug_log() {
 }
 
 initialize_debug() {
+    local system_uptime
     # Clear the screen so the debug log is readable
     clear
     show_disclaimer
     # Display that the debug process is beginning
     log_write "${COL_PURPLE}*** [ INITIALIZING ]${COL_NC}"
     # Timestamp the start of the log
     log_write "${INFO} $(date "+%Y-%m-%d:%H:%M:%S") debug log has been initialized."
+    # Uptime of the system
+    system_uptime=$(uptime | awk -F'( |,|:)+' '{if ($7=="min") m=$6; else {if ($7~/^day/){if ($9=="min") {d=$6;m=$8} else {d=$6;h=$8;m=$9}} else {h=$6;m=$7}}} {print d+0,"days,",h+0,"hours,",m+0,"minutes"}')
```

Comment:
Why do we need such a complex processing of the output?
Anyway, please add a comment about where you got this from. I guess one of the answers from

https://stackoverflow.com/questions/56669767/escaping-complicated-command-line-arguments-in-bash-and-ssh

?

Reply:
> Re-use the uptime code from PADD

Reply:
Tracked it down to
https://github.com/pi-hole/pi-hole/pull/368

Their source was 
https://stackoverflow.com/questions/28353409/bash-format-uptime-to-show-days-hours-minutes

Can add a note to the comment

Reply:
Yeah, it may just being nit-picky, but it's always nice to respect (original) authorship.